migrations/tasks/rewards-flip-negative-costs.js

```diff
@@ -0,0 +1,57 @@
+// @migrationName = 'RewardsMigrationFlipNegativeCostsValues';
+// @authorName = 'hamboomger';
+// @authorUuid = '80b61b73-2278-4947-b713-a10112cfe7f5';
+
+/*
+ * For each reward with negative cost, make it positive
+ * by assigning it an absolute value of itself
+ */
+
+import { Task } from '../../website/server/models/task';
+
+async function flipNegativeCostsValues () {
+  const query = {
+    type: 'reward',
+    value: { $lt: 0 },
+  };
+
+  const totalCount = await _getTotalCount(query);
+  console.log(`Total documents to be changed: ${await Task.find().exec()}`);
+  if (totalCount === 0 || totalCount > 1) {
+    return;
+  }
+
+  let skip = 0;
+  const limit = 200;
+
+  while (skip < totalCount) {
```

Comment:
skip and limit work but tend to have some serious performance problems when delaing with large number of documents, what we usually do is sort by `_id` and then pick only the results above the last `_id`, you can see an example here https://github.com/HabitRPG/habitica/blob/develop/migrations/archive/2020/20200131_habit_birthday.js#L71-L86

Reply:
@paglias I pushed new changes, can you take a look pls?

Reply:
@hamboomger thanks for the update and yes I'll give a look at all updated PRs tomorrow :) Unfortunately we're a small team, I'm the only one reviewing PRs and I don't work full time on this project so I try to review PRs every Monday. Thanks again for the PRs, it's very appreciated :)